nix/ignis.nix

```diff
@@ -34,6 +42,13 @@ pkgs.stdenv.mkDerivation {
     pkgs.dart-sass
   ];
 
+  postUnpack = ''
+    pushd $sourceRoot
+    mkdir -p subprojects
+    cp -rf --no-preserve=mode ${gvc} ./subprojects/gvc
```

Comment:
I think you are copying the wrong folder here. 
`cp -rf --no-preserve=mode ${gvc}/. ./subprojects/gvc` works fine.

Reply:
Thank you for the information :+1: